onnxruntime/core/providers/nnapi/nnapi_builtin/builders/op_builder.cc

```diff
@@ -620,7 +621,10 @@ Status BinaryOpBuilder::AddToModelBuilderImpl(ModelBuilder& model_builder, const
   } else if (op_type == "Pow") {
     add_activation = false;  // ANEURALNETWORKS_POW does not have activation
     op_code = ANEURALNETWORKS_POW;
-  } else {
+  } else if (op_type == "PRelu") {
+    add_activation = false;  // ANEURALNETWORKS_PRELU does not have activation
+    op_code = ANEURALNETWORKS_PRELU;
+    } else {
```

Comment:
not sure about this but - Does the second input of PRelu op, `slope` needs any special handling as an initializer? or can it just share the binaryopbuilder

Reply:
The NNAPI [spec](https://developer.android.com/ndk/reference/group/neural-networks#group___neural_networks_1ggaabbe492c60331b13038e39d4207940e0a82e7e7c709ad117a0178203217be30cf) doesn't require any special scaling etc. of the slope input, and allows the scale and zp to differ from the first input. Based on that, there's no processing we need to do ahead of time, so I don't think it should matter whether slope comes from an initializer or not.

Reply:
ah ok - I see now. this was a question actually I was wondering before for initializers. thx for the explanation.